rollup.config.js

```diff
@@ -1,17 +1,30 @@
+/** @format */
 import typescript from 'rollup-plugin-typescript2';
-import { nodeResolve } from "@rollup/plugin-node-resolve";
-import { terser } from "rollup-plugin-terser";
+import {
+  nodeResolve
+} from "@rollup/plugin-node-resolve";
+import autoNamedExports from 'rollup-plugin-auto-named-exports';
+import {
+  terser
```

Comment:
one line?

Reply:
Will leave this for now as I can't quite figure out which prettier option would fix this.